### PR TITLE
Set up Components for UI development

### PR DESCRIPTION
**Pull request by yin-ka**
In this project I have Successfully:

- Delete all the boilerplate.
- Create a new directory called components
- Create a new MyCalculator.js file.
- Create a React component.
- Add child component.
- Imported and used Components created.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -37,3 +37,50 @@
...
+}
+
+.calc-display {
+  width: 100%;
+  background-color: #7a7c8e;
+  color: #fff;
+  height: 40px;
+  text-align: right;
+  font-size: 28px;
+}
```
Comment (Ghiftee) [inline src/App.css:53]: - [ ] Good job deleting the CRA boilerplate files in the public folder, however, there are still some left in your app as shown below, please delete them
Screenshot 2023-07-06 at 14 55 52

File: src/components/Button.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (Ghiftee) [inline src/components/Button.js]: - [ ] Kindly do not disable linters as it helps us write cleaner code

File: src/components/MyCalculator.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import Button from './Button';
+
+function MyCalculator() {
```
Comment (Ghiftee) [inline src/components/MyCalculator.js]: - [ ] According to the instructions, the name of this component should be `Calculator.js` as shown in the requirements below, please let's try to follow project requirements as it is best practice for software developers.
Screenshot 2023-07-06 at 14 51 31

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello @yin-ka 👋🏽 

Good job working on the project so far! 
To highlight 👍🏽 
✔️ New calculator component is created
✔️ `Calculator.js` has a child component
✔️ Component matches wireframe design

However, there are some issues that you need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review 👇🏽 _

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/App.css (hunk 1 above)
Comment (yin-ka) [reply src/App.css:53]: Okay, thanks.

**Review by joel-kalema — APPROVED** [review]
Hi @yin-ka 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 


## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**